Frameworks/UIKit/UIView.mm

```diff
@@ -2252,14 +2251,18 @@ - (CGPoint)convertPoint:(CGPoint)pos fromView:(UIView*)fromView {
 */
 - (void)drawLayer:(CALayer*)layer inContext:(CGContextRef)context {
     UIGraphicsPushContext(context);
+    _CGContextPushBeginDraw(context);
+
+    auto popEnd = wil::ScopeExit([context]() {
+        _CGContextPopEndDraw(context);
+        UIGraphicsPopContext();
+    });
 
     CGRect bounds;
     // TODO(DH)
     bounds = self.bounds;
-    //bounds = CGContextGetClipBoundingBox(context);
+    // bounds = CGContextGetClipBoundingBox(context);
```

Comment:
@DHowett-MSFT is taking care of this

---
In reply to: [97467495](https://github.com/Microsoft/WinObjC/pull/1750#discussion_r97467495) [](ancestors = 97467495)